cpp/src/groupby/hash/groupby.cu

```diff
@@ -133,7 +228,7 @@ flatten_single_pass_aggs(std::vector<aggregation_request> const& requests)
  * @see groupby_null_templated()
  */
 void sparse_to_dense_results(std::vector<aggregation_request> const& requests,
```

Comment:
This method needs access to the hash map because `std` cannot be calculated by its simple aggregations alone. `std` uses the simple aggs to calculate the result while accessing all the elements of the column again. This means the value columns and hash map need to be sent to the visitor `hash_compound_agg_finalizer` as well.

Reply:
Done. Added STD, VARIANCE, SUM_OF_SQUARES.